fixed minizip url

ttomasz opened the pull request:
When trying to build I got 404 error. It appears that the url changed. I see all versions under `/fossils`. We could also use GitHub release link.

aleixpol commented:
Trying a different approach here by adding x-checker-data, so it will update when the latest stable version changes.

ttomasz commented:
@aleixpol i don't think your solution is working properly I updated my fork and tried to build but it still throws 404. They released 1.3.1 and removed 1.3 from the main page... Why not just use fossils endpoint?
